akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -32,7 +33,7 @@ import org.scalatest.concurrent.Eventually
  */
 class Http2ClientSpec extends AkkaSpecWithMaterializer("""
     akka.http.server.remote-address-header = on
-    akka.http.server.http2.log-frames = on
+    akka.http.client.http2.log-frames = on
   """)
```

Comment:
I've enabled logging of frames on the client. Not sure about `server.remote-address-header`, though.

Reply:
Can be removed I guess.